vnext/Shared/OInstance.cpp

```diff
@@ -418,6 +422,25 @@ InstanceImpl::InstanceImpl(
           m_devSettings->jsiRuntimeHolder =
               std::make_shared<Microsoft::JSI::ChakraRuntimeHolder>(m_devSettings, m_jsThread, nullptr, nullptr);
           break;
+        case JSIEngineOverride::V8Napi: {
+#if defined(USE_V8)
+          std::unique_ptr<facebook::jsi::ScriptStore> scriptStore = nullptr;
```

Comment:
>= nullptr [](http://example.com/codeflow?start=66&length=9)

(nit) many guidances prefer leaving out such assignments when well defined default is available